modules/programs/tmux.nix

```diff
@@ -277,6 +473,79 @@ in
         '';
       })
 
+      (mkIf (cfg.status != null) {
+        home.file.".tmux.conf".text = mkOrder 1000 ''
+          set -g status on
+
+          set -g status-interval ${builtins.toString cfg.status.updateInterval}
+          set -g status-justify ${cfg.status.justification}
+          set -g status-left-length ${builtins.toString cfg.status.leftLength}
+          set -g status-right-length ${builtins.toString cfg.status.rightLength}
+
+          ${optionalString (cfg.status.currentWindowFormat != "")
+                           "setw -g window-status-current-format '${cfg.status.currentWindowFormat}'"}
+          ${optionalString (cfg.status.windowFormat != "")
+                           "setw -g window-status-format '${cfg.status.windowFormat}'"}
+          ${optionalString (cfg.status.leftFormat != "")
+                           "set -g status-left '${cfg.status.leftFormat}'"}
+          ${optionalString (cfg.status.rightFormat != "")
+                           "set -g status-right '${cfg.status.rightFormat}'"}
+          ${optionalString (cfg.status.style != "")
+                           "set -g status-style '${cfg.status.style}'"}
+          ${optionalString (cfg.status.windowStyle != "")
+                           "setw -g window-status-style '${cfg.status.windowStyle}'"}
+          ${optionalString (cfg.status.currentWindowStyle != "")
+                           "setw -g window-status-current-style '${cfg.status.currentWindowStyle}'"}
+          ${optionalString (cfg.status.messageStyle != "")
+                           "setw -g message-style '${cfg.status.messageStyle}'"}
+        '';
+      })
+
+      (mkIf (cfg.status == null) {
```

Comment:
Why not simply add these options to the `tmuxConf` variable?

Reply:
I'm thinking of finally move it all to `tmuxConf`, this game of priorities looks awkward.